Arrangement_on_surface_2/include/CGAL/Arr_dcel_base.h

```diff
@@ -760,7 +768,7 @@ class Arr_outer_ccb : public In_place_list_base<Arr_outer_ccb<V,H,F> > {
  * Representation of an inner CCB.
  */
 template <class V, class H, class F>
-class Arr_inner_ccb : public In_place_list_base<Arr_inner_ccb<V,H,F> >
+class Arr_inner_ccb : public Compact_container_base
```

Comment:
You can use pointer `p_f ` instead of using Compact_container_base (you will save one pointer per element).

Reply:
You are right indeed.

The reason I did not use it is that it will already be squatted when https://github.com/CGAL/cgal/pull/5166/files is merged (as I'm working on an experiment branch with all optimizations, I did not modify this part). I could modify this PR and revert back in the other one, but it's probably a waste of time.